src/main/scala/io/sdkman/changelogs/JoernMigrations.scala

```diff
@@ -0,0 +1,26 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "071")
+class JoernMigrations {
+
+  @ChangeSet(order = "001", id = "001_add_joern_1_1_524", author = "ursachec")
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "io.joern",
```

Comment:
This is the identifier of the candidate, and we typically don't have special characters. I would suggest the simple `joern` or otherwise `joerncli` if you have more tools available.